src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="field_table">
+        <legend>Personal information:</legend>
+        <label>
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            step="1"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="field__info">
+          Name:
+          <input
+            type="text"
+            name="name"
+            step="2"
+            autocomplete="off"
+          >
+        </label>
+        <label class="field__info">
+          How old are You?
+          <input
+            type="number"
+            value="12"
+            name="old"
+            step="3"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="field__info">
```

Comment:
Here as well, for example. I will not point to every place in you code, you should do it by yourself 
```suggestion
        </label>
        
        <label class="field__info">
```